fiftyone/utils/sam3.py

```
@@ -1156,49 +1155,129 @@ def _forward_pass_concept(self, video_reader, sample):
+                            text_prompt,
+                            sample_detections,
+                        )
+
+            else:
+                # Exemplar-only mode (no text prompts): run every label on
+                # every frame without filtering.
+                for fi, frame_data in sorted(
+                    self._curr_exemplar_prompts.items()
+                ):
+                    for label, label_data in frame_data.items():
+                        text_label = label if label is not "visual" else None
```

Ruff (0.15.20)

[error] Use `!=` to compare constant literals

Replace `is not` with `!=`

(F632)